Makefile

```diff
@@ -0,0 +1,52 @@
+TESTS        ?= ./...
+
+bin          := accounts-statistics-tool
+lint_output  := lint.txt
+
+commit       := $(shell git rev-parse --short HEAD)
+tag          := $(shell git tag -l 'v*-rc*' --points-at HEAD)
+version      := $(shell if [[ -n "$(tag)" ]]; then echo $(tag) | sed 's/^v//'; else echo $(commit); fi)
+
+.EXPORT_ALL_VARIABLES:
+GO111MODULE = on
+
+.PHONY: all
+all: build
+
+.PHONY: fmt
+fmt:
+	go fmt ./...
+
+.PHONY: build
+build: fmt
+	go build
```

Comment:
At present this doesn't specify an 'output' name, so the resulting binary is just named according to the project: `innovation-accounts-statistics-tool`, however in other parts of this Makefile we're expecting the binary to be called `accounts-statistics-tool`. To rectify this, change this line to: 
```
go build -o ./$(bin)
```

Reply:
Awesome catch!